src/Reflection/Annotated.agda

```diff
@@ -0,0 +1,335 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Annotated reflected syntax.
+--
+-- NOTE: This file does not check under --without-K due to restrictions
+--       in the termination checker. In particular recursive functions
+--       over a universe of types is not supported by --without-K.
+------------------------------------------------------------------------
+
+{-# OPTIONS --safe --with-K #-}
+
+module Reflection.Annotated where
+
+open import Level                               using (Level; 0ℓ; suc; _⊔_)
+open import Category.Applicative                using (RawApplicative)
+open import Data.Bool                           using (Bool; false; true; if_then_else_)
```

Comment:
Again, importing the `Base` modules will prevent us from dragging in too many other files.